sources/packages/web/src/assets/css/vuetify.scss

```diff
@@ -169,3 +169,10 @@
 .v-overlay-container .v-tooltip .v-overlay__content {
   @extend .tooltip-style;
 }
+
+// Changes the background of readonly inputs.
+.v-input--readonly {
+  .v-field {
+    background-color: #e4e6eb;
```

Comment:
mm, I know you might have added this as per figma. but wondering if shouldn't we follow our figma colors https://www.figma.com/file/9Om2tzLAVEoMQrjmrH4fhu/MOD-SIMS?node-id=1%3A494&mode=dev

Reply:
I guess it should, I will use the below one.
![image](https://github.com/bcgov/SIMS/assets/61259237/564b32e5-48bb-4201-a31c-8023f66b502e)

Reply:
Used the existing variable for the same.